ui_framework/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["react", "@spalger/babel-presets"]
-}
```

Comment:
Oh, `start`, I thought it was for testing.

Yeah, so that's "worrying" as it's running different babel config based on ui framework development and Kibana development (and production), as those will be built using Kibana's babel config. I put it back for now, though, then we can handle that in a later PR.